start_test_network.sh

```diff
@@ -18,5 +18,4 @@ source env/bin/activate
 pip install -q -U -r ../tests/requirements.txt
 echo "Python environment successfully setup"
 
-# TODO: https://github.com/microsoft/CCF/issues/612
-python ../tests/start_network.py --package "$1" --label test_network
\ No newline at end of file
+CURL_CLIENT=ON python ../tests/start_network.py --package "$1" --label test_network
```

Comment:
Does this work? The Python needs both `HTTP` and `CURL_CLIENT` to be in the env - afaict we're still relying on a user setting `HTTP`?

Reply:
Yep. `CURL_CLIENT=ON` has no effect if `HTTP` is not set. I think it is OK for now, even though it can be a little confusing for users looking at this script when building CCF with framed JSON-RPC. What do you think?

Reply:
I think this is fine.

Perhaps we should be treating this as a CMake-configured file, and substituting in the environment vars determined by the CMake build args, but this isn't worth investigating for now.